go.mod

```diff
@@ -568,7 +568,6 @@ replace (
 	github.com/spf13/viper => github.com/spf13/viper v1.11.0 // required to drop dependency on deprecated github.com/coreos/etcd and github.com/coreos/go-etcd
 	github.com/valyala/fasthttp => github.com/valyala/fasthttp v1.36.0
 	golang.org/x/crypto => golang.org/x/crypto v0.5.0
-	golang.org/x/net => golang.org/x/net v0.7.0
```

Comment:
Given the go.sum changes I'm not sure if v0.17.0 is the winning candidate. Mind updating the replace to enforce it for our builds?